src/components/SplitButton/SplitButton.jsx

```diff
@@ -0,0 +1,215 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, filterTypes, getFirst, omitProps }  from '../../util/component-types';
+import Button from '../Button/Button';
+import ButtonGroup from '../ButtonGroup/ButtonGroup';
+import CaretIcon from '../Icon/CaretIcon/CaretIcon';
+import DropMenu from '../DropMenu/DropMenu';
+import * as reducers from './SplitButton.reducers';
+
+const cx = lucidClassNames.bind('&-SplitButton');
+
+const {
+	any,
+	bool,
+	func,
+	oneOf,
+	shape,
+	string,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["controls", "buttons"], "madeFrom": ["Button", "DropMenu"]}
+ *
+ * SplitButtons allow you to combine a single main `Button` together with a
+ * list of additional Buttons with actions which will be rendered within a
+ * DropMenu.
+ */
+const SplitButton = createClass({
+	displayName: 'SplitButton',
+
+	components: {
+		/**
+		 * One of many potential `Button`s to render in this `SplitButton`. The
+		 * first `Button` will be used as the Primary button, while all others will
+		 * be rendered within the `DropMenu` below.
+		 */
+		Button: createClass({
+			displayName: 'SplitButton.Button',
+			propTypes: {
+				/**
+				 * The children to render within the `Button`.
+				 */
+				children: any,
```

Comment:
do you really want `any` here, or is `node` a little more appropriate? even tho both are kind of ambiguous.